.buildkite/pipeline.yml

```diff
@@ -4,6 +4,8 @@ steps:
       - mmake lint
       - mmake dynamo.is-up
       - mmake test
+    agents:
+      queue: search
```

Comment:
Did you need this for a stronger instance type? cc @caiokf

Reply:
yes. otherwise elastic search wouldn't go up

Reply:
@caiokf I was asking because this is not related to elasticsearch, this is the ES module. I wonder if we should reconsider naming CI agent queues per instance types instead of services.

Reply:
ha. true. sorry i got confused. I don't remember any reason why es should have a bigger instance. maybe we could remove this

Reply:
Yeah, I had to bump up this instance as I added [localstack](https://hub.docker.com/r/localstack/localstack) to help testing the SNS notification piece. I wasn't comfortable assuming that the payload was "just going to work"… 🤷‍♂😄

Reply:
In this case, I think we could have agent queues be named after instance sizes. Before it didn't make sense because there was specific elastic search config going on, docker-compose, etc. Now there are no such things.